### PR TITLE
Use universal-analytics to collect usage stats.

### PR DESCRIPTION
**Pull request by nya3jp**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,6 +48,10 @@ const App = () => (
 
 ReactDOM.render(<App />, document.getElementById('root'));
 
+const visitor =
+      UniversalAnalytics(constants.GOOGLE_ANALYTICS_TRACKING_ID, {https: true});
```
Comment (hidehikoabe) [inline src/index.js]: style: 4 indent.

File: src/constants.js [hunk 2]
```diff
@@ -26,3 +27,6 @@ export const PRESETS = [
     type: 'gerrit',
   },
 ];
+
+// Google Analytics Tracking ID.
```
Comment (hidehikoabe) [inline src/constants.js]: Optional: How about making this to the Jsdoc style? 

/*\* Google analytics Tracking ID */

**Comment by hidehikoabe** [comment]
The change LGTM with minor comments.

BTW, what's the motivation to switch it from Google Analytics dedicated script to the third party one?
Could you clarify in the commit log?

**Comment by nya3jp** [comment]
Addressed comments and added description. Thanks!

**Pull request merged**